stackit/internal/services/objectstorage/bucket/resource.go

```diff
@@ -94,7 +94,7 @@ func (r *bucketResource) Schema(_ context.Context, _ resource.SchemaRequest, res
 		"main":                     "ObjectStorage bucket resource schema.",
 		"id":                       "Terraform's internal resource identifier. It is structured as \"`project_id`,`bucket_name`\".",
 		"bucket_name":              "The bucket name. It must be DNS conform.",
-		"project_id":               "STACKIT Project ID to which the bucket is associated.",
+		"project_id":               "STACKIT Project ID to which the bucket is associated to.",
```

Comment:
Either
- "STACKIT Project ID to which the bucket is associated."
- "STACKIT Project ID which the bucket is associated to."

I think the 1st is correct, but not sure

Reply:
Yeah I guess its not the perfect english but yeah it works